sdk/eventhubs/azure_messaging_eventhubs/src/consumer/mod.rs

```diff
@@ -189,10 +190,9 @@ impl ConsumerClient {
         Ok(())
     }
 
-    /// Receives events from a specific partition of the Event Hub.
+    /// Attaches a message receiver to a specific partition of the Event Hub.
     ///
-    /// This function establishes a connection to the specified partition of the Event Hub and starts receiving events from it.
-    /// It returns a stream of `ReceivedEventData` items, representing the events received from the partition.
+    /// This function establishes a connection to the specified partition of the Event Hub and returns a MessageReceiver which can be used to receive messages from it.
```

Comment:
Maybe in a future PR, but link types as much as possible like I showed in my first couple of comments. Makes for better user docs.